ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -56,9 +61,15 @@ private[index] class BuffersUpdater(
           randomFactor = 0.0,
         )
       )(() => subscribeToTransactionLogUpdates(updaterIndex.get))
-      .map { case ((offset, eventSequentialId), update) =>
-        updateCaches(offset, update)
-        updaterIndex.set(Some(offset -> eventSequentialId))
```

Comment:
The updaterIndex is no longer set anywhere. This means that re-subscription to the upstream source is no longer possible

Reply:
Indeed. It was a conscious decision to leave this detail out in the first iteration.

Reply:
You should leave a TODO then, it is easily overlooked

Reply:
Good point.